docs/reference-architectures/identity/adds-extend-domain-content.md

```diff
@@ -69,7 +69,7 @@ Deploy the VMs running AD DS into at least two [Availability Zones](/azure/avail
 
 ## Manageability considerations
 
-Perform regular AD DS backups. Don't copy the VHD files of domain controllers instead of performing regular backups, because the AD DS database file on the VHD may not be in a consistent state when it's copied, making it impossible to restart the database.
+Perform regular AD DS backups. Don't copy the VHD files of domain controllers instead of performing regular backups, because the AD DS database file on the VHD may not be in a consistent state when it's copied, making it impossible to restart the database. You can use [Azure Backup](/azure/backup/active-directory-backup-restore#backing-up-azure-vm-domain-controllers) to back up domain controller VMs.. 
```

Comment:
```suggestion
Perform regular AD DS backups. Don't copy the VHD files of domain controllers instead of performing regular backups, because the AD DS database file on the VHD may not be in a consistent state when it's copied, making it impossible to restart the database. You can use [Azure Backup](/azure/backup/active-directory-backup-restore#backing-up-azure-vm-domain-controllers) to back up domain controller VMs.
```